src/AppInstallerCLICore/ConfigureExportCommand.cpp

```diff
@@ -39,26 +43,30 @@ namespace AppInstaller::CLI
     {
         context <<
             VerifyIsFullPackage <<
+            SearchSourceForPackageExport <<
             CreateConfigurationProcessor <<

```

Comment:
```suggestion
            CreateConfigurationProcessorWithoutFactory <<
```
The current code launches another process for the factory and then never uses it (at least from my reading).  This will speed things up a lot.

Reply:
Discussed with John, we'll leave it as is. Other configuration resources need it and we may also explore exporting configurations related to exported packages later.